content/en/tutorials/cloud-pods-collaborative-debugging/index.md

```diff
@@ -0,0 +1,442 @@
+--header 'Content-Type: application/json' \
+--data '{
+  "id": "34534",
+  "name": "EcoFriendly Water Bottle",
+  "description": "A durable, eco-friendly water bottle designed to keep your drinks cold for up to 24 hours and hot for up to 12 hours. Made from high-quality, food-grade stainless steel, it'\''s perfect for your daily hydration needs.",
+  "price": "29.99"
+}'
+
+$ curl --location "http://$rest_api_id.execute-api.localhost.localstack.cloud:4566/dev/productApi?id=82736" \
+--header 'Content-Type: application/json' \
+--data '{
+  "id": "82736",
+  "name": "Sustainable Hydration Flask",
+  "description": "This sustainable hydration flask is engineered to maintain your beverages at the ideal temperature—cold for 24 hours and hot for 12 hours. Constructed with premium, food-grade stainless steel, it offers an environmentally friendly solution to stay hydrated throughout the day.",
+  "price": "31.50"
+}'
+```
+
+The response is the one that we expect: `Product added/updated successfully.`
+
+However, retrieving one of the products does not return the desired result:
+
+{{< command >}}
+$ curl --location "http://$rest_api_id.execute-api.localhost.localstack.cloud:4566/dev/productApi?id=34534"
+{{</ command >}}
+
+```bash
+Internal server error⏎  
+```
+
+An `Internal server error⏎`  does not give out too much information. The new colleague doesn’t know for sure what could be 
+causing this. The Lambda code looks fine, the configurations also look ok.
+
+# **Using Cloud Pods for fast collaborative debugging**
+
+## 1. Creating a Cloud Pod
+
+To share this exact environment and issue with a teammate, the new colleague only needs to run a simple `localstack pod` command:
+
+{{< command >}}
+$ localstack pod save cloud-pod-product-app
+{{</ command >}}
+
+```bash
+Cloud Pod `cloud-pod-product-app` successfully created ✅
+Version: 1
+Remote: platform
+Services: sts,iam,apigateway,dynamodb,lambda,s3,cloudwatch,logs
+```
+
+LocalStack provides a remote storage backend that can be used to store the state of your application and share it with your team members.
+
+The Cloud Pods CLI is included in the LocalStack CLI installation, so there’s no need for additional plugins to begin using it. 
+The `LOCALSTACK_AUTH_TOKEN` needs to be set as an environment variable.
+
+Additionally, there are other commands for managing Cloud Pods included in the CLI:
+
+```bash
+$ localstack pod --help
+Usage: localstack pod [OPTIONS] COMMAND [ARGS]...
+
+  Manage the state of your instance via Cloud Pods.
+
+Options:
+  -h, --help  Show this message and exit.
+
+Commands:
+  delete    Delete a Cloud Pod
+  list      List all available Cloud Pods
+  load      Load the state of a Cloud Pod into the application runtime/ Users can import Cloud Pods from...
+  remote    Manage cloud pod remotes
+  save      Create a new Cloud Pod
+  versions  List all available versions for a Cloud Pod
```

Comment:
`for` is colored differently. Any idea why? Maybe we should use `console` or `shell`?